src/PerformanceTests/Tests/Categories/Base.cs

```diff
@@ -76,6 +76,7 @@ protected void Tasks(Permutation permutation, [CallerMemberName] string memberNa
         {
             var fixtureType = GetType();
             var fixture = fixtureType.GetCustomAttribute<TestFixtureAttribute>();
+            TestContext.WriteLine($"Running test {fixtureType} ({TestContext.CurrentContext.Test.FullName}/{TestContext.CurrentContext.Test.MethodName}");
```

Comment:
that has turned out to be a very useful information when investigating failing tests as `dotnet test` only shows the failing test method, which can't easily be aligned with the failing test fixture.